mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -440,6 +440,21 @@ func resourceContainerCluster() *schema.Resource {
 										Description: `The Google Cloud Platform Service Account to be used by the node VMs.`,
 									},
 									<% unless version == 'ga' -%>
+									"image_type": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Default:     "cos",
+										Description: `The default image type used by NAP once a new node pool is being created.`,
+										ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
```

Comment:
We have the [`validation.StringInSlice`](https://github.com/hashicorp/terraform-plugin-sdk/blob/112e2164c381d80e8ada3170dac9a8a5db01079a/helper/validation/strings.go#L132) function available that we typically use. Would that work here?

Reply:
Yes, thanks for the suggestion. It would be better than the piece of code that I wrote for validation. I'll update that.